src/Symfony/Component/Security/Core/Authorization/Voter/RoleVoter.php

```diff
@@ -57,6 +58,10 @@ public function vote(TokenInterface $token, $object, array $attributes)
         $roles = $this->extractRoles($token);
 
         foreach ($attributes as $attribute) {
+            if ($attribute instanceof RoleInterface) {
+                $attribute = $attribute->getRole();
```

Comment:
nevermind, that's already covered in `supportsAttribute()` via the `is_string()` check.